### PR TITLE
WIP / Draft PRs check

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation

Checks that no PR has WIP in the title or body.
Also, checks that draft PRs are not made.
## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by jameesjohn** [comment]
PTAL @ankita240796.
Thanks!

File: index.js [hunk 1]
```diff
@@ -59,4 +63,13 @@ module.exports = (oppiabot) => {
         context);
     }
   });
+
+  oppiabot.on('pull_request.edited', async context => {
+    if (whitelistedAccounts.includes(context.repo().owner.toLowerCase()) &&
```
Comment (ankita240796) [inline index.js:68]: Break after opening bracket.

File: lib/checkWipDraftPR.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      'ready to be reviewed and ensure that it is without any WIP text. ' +
+      'Thanks!');
+    const commentParams = context.repo({
+      issue_number: pullRequest.number,
+      body: commentBody,
+    });
+    await context.github.issues.createComment(commentParams);
+
+    // Close Pull Request.
+    const closeIssueParams = context.repo({
```
Comment (ankita240796) [inline lib/checkWipDraftPR.js:46]: closePRParams

File: lib/checkWipDraftPR.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  const prAuthor = pullRequest.user.login;
+
+  if (isDraftPr(pullRequest) || isWIPPr(pullRequest)) {
+    const link = 'here'.link(
+      'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia' +
+      '#wip--draft-pull-requests');
+    // Comment on Pull Request.
+    const commentBody = (
+      'Hi @' + prAuthor + ', WIP/Draft PRs are highly discouraged. You can ' +
+      'learn more about it ' + link + '. Do well to reopen it when it\'s ' +
```
Comment (ankita240796) [inline lib/checkWipDraftPR.js:36]: You can reopen it .... and ensure that it is without any WIP phrase in title or body.

File: index.js (hunk 1 above)
Comment (jameesjohn) [reply index.js]: Done.

File: lib/checkWipDraftPR.js (hunk 3 above)
Comment (jameesjohn) [reply lib/checkWipDraftPR.js]: Done.

File: lib/checkWipDraftPR.js (hunk 2 above)
Comment (jameesjohn) [reply lib/checkWipDraftPR.js]: Done.

File: spec/checkWipDraftPRSpec.js [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+
+    it('creates comment for WIP PRs', () => {
+      expect(github.issues.createComment).toHaveBeenCalled();
+      expect(github.issues.createComment).toHaveBeenCalledTimes(1);
+
+      // Comment on Pull Request.
+      const link = 'here'.link(
+        'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia' +
+        '#wip--draft-pull-requests');
+      const author = pullRequestEditedPayload.payload.pull_request.user.login;
```
Comment (ankita240796) [inline spec/checkWipDraftPRSpec.js:83]: Are we doing these tests only for edited case, why not open and re-opened as well?

Comment (jameesjohn) [reply spec/checkWipDraftPRSpec.js:92]: Added test for opened and reopened.

**Comment by jameesjohn** [comment]
PTAL @ankita240796. Thanks!.

**Comment by ankita240796** [comment]
Deployed this change as well! Congrats on introducing a new check!

**Comment by jameesjohn** [comment]
Yay! Thanks!

**Pull request merged**